ios/FluentUI.Demo/FluentUI.Demo/Demos/ActivityIndicatorDemoController.swift

```diff
@@ -0,0 +1,232 @@
+        return ActivityIndicatorDemoSection.allCases[section].rows.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let row = ActivityIndicatorDemoSection.allCases[indexPath.section].rows[indexPath.row]
+
+        switch row {
+        case .hidesWhenStopped:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: BooleanCell.identifier) as? BooleanCell else {
+                return UITableViewCell()
+            }
+            cell.setup(title: row.title, isOn: self.shouldHideWhenStopped)
+            cell.titleNumberOfLines = 0
+            cell.onValueChanged = { [weak self, weak cell] in
+                self?.shouldHideWhenStopped = cell?.isOn ?? true
+            }
+            return cell
+        case .startStopActivity:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: ActionsCell.identifier) as? ActionsCell else {
+                return UITableViewCell()
+            }
+
+            cell.setup(action1Title: row.title)
+            cell.action1Button.addTarget(self,
+                                         action: #selector(startStopActivity),
+                                         for: .touchUpInside)
+            cell.bottomSeparatorType = .full
+            return cell
+        case .swiftUIDemo:
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: TableViewCell.identifier) as? TableViewCell else {
+                return UITableViewCell()
+            }
+            cell.setup(title: row.title)
+            cell.accessoryType = .disclosureIndicator
+
+            return cell
+        case.demoOfSize:
+            let cell = TableViewCell()
+
+            let activityIndicatorSize = MSFActivityIndicatorSize.allCases.reversed()[indexPath.row]
+            let activityIndicatorDictionaries = [defaultColorIndicators, customColorIndicators]
+            let activityIndicatorPath = indexPath.section - 2
+            let activityIndicator = activityIndicatorDictionaries[activityIndicatorPath][activityIndicatorSize]
+
+            cell.setup(title: activityIndicatorSize.description,
+                       customView: activityIndicator?.view)
+            cell.titleNumberOfLines = 0
+            return cell
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return ActivityIndicatorDemoSection.allCases[section].title
+    }
+
+    override func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
+        return ActivityIndicatorDemoSection.allCases[indexPath.section].rows[indexPath.row] == .swiftUIDemo
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let cell = tableView.cellForRow(at: indexPath) else {
+            return
+        }
+
+        cell.setSelected(false, animated: true)
```

Comment:
Instead of explicit deselection, just set the selection style to `.none` on creation:

```swift
cell.selectionStyle = .none
```

Reply:
We do that to create the fading animation on the background on selecting and deselecting, otherwise the animation of the background won't be there.


https://user-images.githubusercontent.com/68076145/133525405-317e1729-1e6d-4f42-95b6-49730fc34199.mov